### DISCUSSION TITLE
KAM distribution

### DISCUSSION THREAD

**Discussion by NovakEvana** [mtex-toolbox/mtex#1702] [Ask Anything] [answered]
Dear Mtex community,

Happy new year to everyone!
Is there any existing built-in function to draw KAM distribution in mtex (similar to the below image)? Thanks for your help!

![image](https://user-images.githubusercontent.com/117882416/210488132-cdf30bfb-1626-47a5-82d6-79847bd420d8.png)

**Comment by Tijmenvermeij** [comment, marked as answer]
Hi,
Best to always first check the documentation: https://mtex-toolbox.github.io/EBSDKAM.html

**Reply by NovakEvana** [reply]
Hi, I did read the documentation. What I'm asking here is the KAM distribution chart, not the KAM map. Thanks.

**Reply by kilir** [reply]
Hi,
have a look at `help histogram` .
Cheers,
Rüdiger

**Reply by NovakEvana** [reply]
Thanks Rudiger!

**Reply by mugikumar** [reply]
Hi Nova 
        can u please share the code to get the average KAM value
